src/Ui/template/wrapper.html

```diff
@@ -1,103 +1,119 @@
+  <div class='infopanel-container'>
+    <div class='infopanel'>
+      <span class='closed-num'>8</span>
+      <div class="message">
+        <span class='line line-1'>8 modified files</span><br><span class='line line-2'>content.json, data.json</span>
+      </div>
+      <a href="#Publish" class="button button-submit">Sign & Publish</a>
+      <a href="#Close" class="close">&times;</a>
+    </div>
+  </div>
 
+  <!-- Loadingscreen -->
+  <div class='loadingscreen'>
+    <a href="/Config" class="loading-config">Config</a>
+    <div class='loading-text console'>
+    </div>
+    <div class="flipper-container">
+      <div class="flipper">
+        <div class="front"></div>
+        <div class="back"></div>
+      </div>
+    </div>
+  </div>
 
-<!-- Site Iframe -->
-<iframe src='about:blank' id='inner-iframe' sandbox="allow-forms allow-scripts allow-top-navigation allow-popups allow-modals allow-presentation allow-pointer-lock allow-popups-to-escape-sandbox {sandbox_permissions}" allowfullscreen="true" webkitallowfullscreen="true" mozallowfullscreen="true" oallowfullscreen="true" msallowfullscreen="true"></iframe>
+  <!-- Site Iframe -->
+  <iframe src='about:blank' id='inner-iframe' sandbox="allow-forms allow-scripts allow-top-navigation allow-popups allow-modals allow-presentation allow-pointer-lock allow-popups-to-escape-sandbox {sandbox_permissions}" allowfullscreen="true" webkitallowfullscreen="true" mozallowfullscreen="true" oallowfullscreen="true" msallowfullscreen="true"></iframe>
 
-<!-- Site info -->
-<script id="script_init" nonce="{script_nonce}">
-iframe_src = "{file_url}{query_string}"
-console.log("Changing url from " + document.getElementById("inner-iframe").src + " to " + iframe_src)
-document.getElementById("inner-iframe").src = document.getElementById("inner-iframe").src  // Workaround for Firefox back button bug
-document.getElementById("inner-iframe").src = iframe_src
-address = "{address}"
-wrapper_nonce = "{wrapper_nonce}"
-wrapper_key = "{wrapper_key}"
-ajax_key = "{ajax_key}"
-postmessage_nonce_security = {postmessage_nonce_security}
-file_inner_path = "{file_inner_path}"
-permissions = {permissions}
-show_loadingscreen = {show_loadingscreen}
-server_url = '{server_url}'
-script_nonce = '{script_nonce}'
+  <!-- Site info -->
+  <script id="script_init" nonce="{script_nonce}">
+    iframe_src = "{file_url}{query_string}"
+    console.log("Changing url from " + document.getElementById("inner-iframe").src + " to " + iframe_src)
+    document.getElementById("inner-iframe").src = document.getElementById("inner-iframe").src // Workaround for Firefox back button bug
+    document.getElementById("inner-iframe").src = iframe_src
+    address = "{address}"
+    wrapper_nonce = "{wrapper_nonce}"
+    wrapper_key = "{wrapper_key}"
+    ajax_key = "{ajax_key}"
+    postmessage_nonce_security = {
+      postmessage_nonce_security
+    }
+    file_inner_path = "{file_inner_path}"
+    permissions = {
+      permissions
+    }
```

Comment:
Excess newlines

Reply:
@imachug file_inner_patch breaks my new empty site. I need to remove it to see the site.